package-lock.json

```diff
@@ -109706,9 +109816,9 @@
       }
     },
     "multiformats": {
-      "version": "9.6.5",
-      "resolved": "https://registry.npmjs.org/multiformats/-/multiformats-9.6.5.tgz",
-      "integrity": "sha512-vMwf/FUO+qAPvl3vlSZEgEVFY/AxeZq5yg761ScF3CZsXgmTi/HGkicUiNN0CI4PW8FiY2P0OLklOcmQjdQJhw=="
+      "version": "9.9.0",
+      "resolved": "https://registry.npmjs.org/multiformats/-/multiformats-9.9.0.tgz",
+      "integrity": "sha512-HoMUjhH9T8DDBNT+6xzkrd9ga/XiBI4xLr58LJACwK6G3HTOPeMz4nB4KJs33L2BelrIJa7P0VuNaVF3hMYfjg=="
```

Comment:
we might want to prevent this change because of #1965

Reply:
specific dependency tracking issue: https://github.com/ipfs/ipfs-webui/issues/2048

Reply:
Sooo... actually we don't need that. The dependencies on `ipfs-geoip` are wrong. Fixed those here: https://github.com/ipfs-shipyard/ipfs-geoip/pull/100

Reply:
left a comment at https://github.com/ipfs/aegir/issues/1104#issuecomment-1301956533 and mentioned it in a peek at ipfs-geoip pr